app/models/manageiq/providers/kubevirt/infra_manager.rb

```diff
@@ -53,6 +53,51 @@ def self.description
     @description ||= ManageIQ::Providers::Kubevirt::Constants::PRODUCT
   end
 
+  def self.params_for_create
+    @params_for_create ||= {
+      :title  => "Configure Kubevirt",
+      :fields => [
+        {
+          :component  => "text-field",
+          :name       => "endpoints.default.server",
+          :label      => "Hostname",
+          :isRequired => true,
+          :validate   => [{:type => "required-validator"}]
+        },
+        {
+          :component  => "text-field",
+          :name       => "endpoints.default.port",
+          :type       => "number",
+          :isRequired => true,
+          :validate   => [
+            {:type => "required-validator"},
+            {:type             => "validatorTypes.MIN_NUMBER_VALUE",
```

Comment:
Not sure if this is right, @Hyperkid123 ?

Reply:
Yes that seems correct.

You can see live example with same use case just different values here if you want to see it live: https://data-driven-forms.org/renderer/validators#numbervaluevalidators

Reply:
It lines up with https://github.com/ManageIQ/manageiq-providers-kubevirt/pull/149/files#diff-ff17b05a7c525eec833f0ddb0134a0d9R75